packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.controller.js

```diff
@@ -93,6 +93,7 @@ export default class BillingMainHistoryCtrl extends ListLayoutHelper.ListLayoutC
 
   $onInit() {
     this.selectedBills = [];
+    this.defaultFilterColumn = 'billId';
```

Comment:
I'm having trouble to understand why updating the `defaultFilterColumn` from `date` to `billId` can fix the search here.

Does it mean that the issue came from the routing part?

Any explanation are welcome.
Thank you!

Reply:
issue comes from https://github.com/ovh/manager/blob/master/packages/manager/modules/ng-layout-helpers/src/list

the defaultFilterColumn doesn't seem to be passed correctly to this component, i don't feel like fixing the layout helper because it looks overly complicated TBH